app/model/sdl/Abstract/data.js

```diff
@@ -1488,6 +1488,16 @@ SDL.SDLModelData = Em.Object.create(
           levelspan: 1
         }
       ]
+    },
+
+    /**
+     * Enum for media Player Indicator
+     */
+    exitCommandsEnum: {
```

Comment:
@mkorniichuk I think it should be enough to have `exitCommands` instead of `exitCommandsEnum` , what do you think?